package.json

```diff
@@ -2,33 +2,37 @@
   "name": "@lob/ui-components",
   "version": "0.0.176",
   "engines": {
-    "node": ">=14.16.1",
-    "npm": ">=7.13.0"
+    "node": ">=14.18.2",
+    "npm": ">=8.3.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/ui-components.umd.js",
+  "module": "./dist/ui-components.es.js",
+  "exports": {
+    ".": {
+      "import": "./dist/ui-components.es.js",
+      "require": "./dist/ui-components.umd.js"
+    }
   },
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "build-library": "vue-cli-service build --target lib --name ui-components src/main.js",
+    "build": "vite build",
     "build-storybook": "build-storybook",
-    "lint": "npm run lint:js && npm run lint:css",
-    "lint-fix": "vue-cli-service lint --fix",
     "lint:css": "stylelint 'src/**/*.vue'",
     "lint:js": "eslint --ext .js,.vue .",
-    "test": "npm run test:ci",
-    "test:ci": "npm run test:unit",
-    "test:unit": "vue-cli-service test:unit",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint-fix": "npm run lint --- --fix",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
+    "test:ci": "npm run coverage",
+    "coverage": "vitest run --coverage",
     "storybook": "start-storybook -p 6006"
   },
-  "main": "./dist/ui-components.umd.min.js",
-  "files": [
-    "dist/*",
-    "src/*"
-  ],
   "dependencies": {
-    "core-js": "^3.6.5",
     "date-fns": "^2.23.0",
     "mitt": "^3.0.0",
-    "vue": "^3.0.0",
+    "vue": "^3.2.31",
```

Comment:
I don't know if you did this or Vite/Vitest did this, but thanks!! seems like a good idea.

Reply:
I did this.  I think I updated all of the vue related libraries to the latest while I had the hood opened here. 👍